examples/local_hugging_face_server.py

```diff
@@ -0,0 +1,89 @@
+"""Example server definition for deploying Huggingface models to a local endpoint.
```

Comment:
can you make a folder called `models/local_hugging_face`, and put this file in there as `server.py`, plus a README describing:
1. what it is (these comments here are fine)
2. what the steps are to run it locally. assume no dependencies are installed (including HF models)
3. How to configure window (e.g. set model to Local, but in this case, that's it)

Reply:
Will do

Reply:
shoot i meant relative to `examples`, so `examples/models/...`